packages/lisk-transactions/src/7_out_transfer_transaction.ts

```diff
@@ -192,22 +192,6 @@ export class OutTransferTransaction extends BaseTransaction {
 			);
 		}
 
-		const transactionExists = store.transaction.find(
-			(transaction: TransactionJSON) =>
-				transaction.id === this.asset.outTransfer.transactionId,
-		);
-
-		if (transactionExists) {
```

Comment:
maybe it was `!transactionExists`?

Reply:
No, I checked the implementation for outTransfer transaction in lisk-core. The check for outTransfer.transactionId was only used for verifying that multiple unconfirmed transactions with same outTransfer.transactionId do not get processed in the same block. (Doesn't make so much, but that's what i get from the implementation).